apps/etterlatte-behandling/src/test/kotlin/IntegrationTest.kt

```diff
@@ -111,7 +112,7 @@ class IntegrationTest : BehandlingIntegrationTest() {
                     BehandlingsBehov(
                         1,
                         Persongalleri("søker", "innsender", emptyList(), emptyList(), emptyList()),
-                        LocalDateTime.now().toString()
+                        Tidspunkt.now().toNorskTidLocalDateTime().toString()
```

Comment:
Er det her ein workaround for å få det til å køyre lokalt?

Reply:
Bare jeg som tenkte at det var fra frontend, fikser til til UTC
d282ae2fd0afdfe7be02a883575ef4a492a419dc